wallet/walletrpc.c

```diff
@@ -797,9 +797,9 @@ static struct command_result *json_listtransactions(struct command *cmd,
 	for (size_t i=0; i<tal_count(txs); i++) {
 		json_object_start(response, NULL);
 		json_add_txid(response, "txid", &txs[i].id);
+		json_add_num(response, "output", txs[i].txindex);
```

Comment:
Not really, this is not the transaction output, rather it's the index of the transaction in the block (second element of the short channel ID).

Reply:
I originally thought so too, but then my tests failed, so I guessed again :(